guides/running-airflow-workflow.md

```diff
@@ -0,0 +1,53 @@
+# Running A Workflow On Airflow
+This guide will walk you through how to deploy a workflow on your existing Airflow cluster.
+
+## Prerequisites
+- The Aqueduct metadata storage must be changed to an AWS S3 bucket. See [here](changing-metadata-store.md) for how to do so. The AWS credentials file for this
+bucket must be copied over to your Airflow cluster.
+- Modify your Airflow cluster's config to use `Basic Authentication`. See the instructions [here](https://airflow.apache.org/docs/apache-airflow/stable/security/api.html#basic-authentication) for how to do so. For now, Aqueduct only supports basic authentication
+using a username and password. We are working on adding support for other authentication methods.
+- Check that Airflow cluster is accessible from the machine running Aqueduct.
+
+## Adding An Airflow Integration
+Once the Airflow cluster has been properly configured and is running, you will need to connect it to Aqueduct via an integration.
+The integration dialog requires the following fields:
+- Name
```

Comment:
I would cut the list here since we have the screenshot. For the S3 credentials, first emphasize that these credentials must have access to the same bucket, then emphasize that they must be on already on the Airflow server. (Maybe even add a note saying Aqueduct cannot do this on your behalf.)